src/components/AppliedFilters.tsx

```diff
@@ -70,6 +70,19 @@ export function AppliedFilters(props: AppliedFiltersProps): JSX.Element {
     hierarchicalFacetsFieldIds
   } = props;
 
+  const duplicateFacets: DisplayableFilter[] = useMemo(() => {
```

Comment:
In this PR, are we only looking for Facets that duplicate selected Static Filters? I'm wondering if that's enough. Could there be other scenarios besides this that result in duplication? For instance, could we have an NLP Filter and a Static Filter that are the same?

Reply:
There shouldn't be any duplicate nlp filters since they are checked against static filters and facets in the prune nlp function of `appliedfilterutils`